controllers/keda/scaledobject_controller.go

```diff
@@ -194,6 +194,14 @@ func (r *ScaledObjectReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 	if err := kedastatus.SetStatusConditions(ctx, r.Client, reqLogger, scaledObject, &conditions); err != nil {
 		r.Recorder.Event(scaledObject, corev1.EventTypeWarning, eventreason.ScaledObjectUpdateFailed, err.Error())
+	}
+
+	if _, present := scaledObject.GetAnnotations()[kedav1alpha1.PausedReplicasAnnotation]; present {
+		ref := scaledObject.Spec.ScaleTargetRef
+		reqLogger.V(1).Info("Detect ScaledObject with pause annotation", "targetKind", ref.Kind, "targetName", ref.Name)
```

Comment:
We try (there are still missing places) to print logs in lower letters
```suggestion
		reqLogger.V(1).Info("detected ScaledObject with pause annotation", "targetKind", ref.Kind, "targetName", ref.Name)
```